messages/de.json

```diff
@@ -20,7 +20,8 @@
     "marketplace": {
       "title": "Marktplatz",
       "market": "Markt",
-      "myPositions": "Meine Positionen"
```

Comment:
We dont really need to keep deutch, can remove from now and in Spanish can just have english. 

When we are complete we will translate it at once. Otherwise will take time from you

Reply:
Okay let me just keep it , going further I will not be updating it anymore